raiden-ts/src/channels/epics/block.ts

```diff
@@ -184,6 +186,37 @@ export function blockStaleEpic(
   );
 }
 
+/**
+ * Fetch settleTimeout from contract once
+ *
+ * @param action$ - Observable of RaidenActions
+ * @param state$ - Observable of RaidenStates
+ * @param deps - RaidenEpicDeps members
+ * @param deps.registryContract - TokenNetworkRegistry instance
+ * @param deps.config$ - Config observable
+ * @param deps.init$ - Observable of sync tasks
+ * @returns Observable of contractSettleTimeout actions
+ */
+export function contractSettleTimeoutEpic(
+  {}: Observable<RaidenAction>,
+  {}: Observable<RaidenState>,
+  { registryContract, config$, init$ }: RaidenEpicDeps,
+): Observable<contractSettleTimeout> {
+  let done$: AsyncSubject<null>;
+  return defer(async () => {
+    done$ = new AsyncSubject();
+    init$.next(done$);
+    return registryContract.callStatic.settle_timeout();
```

Comment:
> Does this basically mean "wait until initialization is done and then make the contract call"?

No. Notice that I'm just pushing the `done$` subject to `init$`, which basically just _registers_ it as an initialization task; `init$` will be complete once all tasks are completed. This specific one is started at startup time, and `done$` is completed after the call succeeds.

Reply:
Also, `latest$` unfortunately needs a value before initialization (because several epics depend on it to even start), so it must have a default value before `init$` completes. We could refactor this to have an init phase and then the normal phase, but this would possibly mean splitting several epics which do mostly the same thing, so I went with the `init$` notifier and default values as initial sync notification mechanism.

Reply:
Okay. Thanks for explaining this!